### PR TITLE
Rename DC to FBC

### PR DESCRIPTION
**Pull request by lipoja**
Since the project was renamed we should also use same names in our code.

### CODE REVIEW

File: tests/test_workers/test_tasks/test_build.py [hunk 1]
```diff
@@ -529,7 +529,7 @@ def test_skopeo_copy_fail_max_retries(mock_run_cmd):
 @mock.patch('iib.workers.tasks.build._add_label_to_index')
 @mock.patch('iib.workers.tasks.build._get_present_bundles')
 @mock.patch('iib.workers.tasks.build.set_registry_token')
-@mock.patch('iib.workers.tasks.build.is_image_dc')
+@mock.patch('iib.workers.tasks.build.is_image_fbc')
 def test_handle_add_request(
     mock_iidc,
```
Comment (chandwanitulsi) [inline tests/test_workers/test_tasks/test_build.py:534]: Minor: Can we rename this to mock_iifbc?

File: tests/test_workers/test_tasks/test_build.py [hunk 2]
```diff
@@ -681,7 +681,7 @@ def side_effect(_, temp_dir, *args, **kwargs):
 
 @mock.patch('iib.workers.tasks.build._cleanup')
 @mock.patch('iib.workers.tasks.build.run_cmd')
-@mock.patch('iib.workers.tasks.build.is_image_dc')
+@mock.patch('iib.workers.tasks.build.is_image_fbc')
 def test_handle_add_request_raises(mock_iidc, mock_runcmd, mock_c):
```
Comment (chandwanitulsi) [inline tests/test_workers/test_tasks/test_build.py:685]: In other tests as well

**Review by chandwanitulsi — COMMENTED** [review]
Minor comment,otherwise LGTM

File: tests/test_workers/test_tasks/test_build.py (hunk 1 above)
Comment (lipoja) [reply tests/test_workers/test_tasks/test_build.py:534]: oh I missed that, thanks!

**Review by chandwanitulsi — APPROVED** [review]

**Pull request merged**